dev/MRTCore/mrt/NuGet/Microsoft.ProjectReunion.nuspec

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+     <metadata minClientVersion="3.3">
+         <id>Microsoft.ProjectReunion</id>
+         <version>1.0</version>
+         <title>$id$</title>
+         <authors>Microsoft</authors>
+         <description>Resource loader for .PRI files.</description>
+         <license type="file">LICENSE.txt</license>
+         <requireLicenseAcceptance>true</requireLicenseAcceptance>
+         <projectUrl>https://aka.ms/MRTCore</projectUrl>
```

Comment:
>https://aka.ms/MRTCore [](start = 21, length = 22)

also need to point to project reunion

Reply:
I'll have somebody register aka.ms/projectreunion to point to github/microsoft/projectreunion.

Reply:
this was already working, can write:  https://aka.ms/ProjectReunion :-)

Reply:
Yep, done!  This file got moved to build/NuSpecs/Microsoft.ProjectReunion.nuspec, where it points to that link now.